.github/workflows/gosec.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Run Gosec Security Scanner
         uses: securego/gosec@v2.10.0
         with:
-          args: -exclude-dir=pkg/promql -exclude-dir=pkg/prompb ./...
+          args: -exclude-dir=pkg/promql -exclude-dir=pkg/prompb -exclude-dir=migration-tool ./...
```

Comment:
Because it was giving false positives for all imports, which made me feel it is not updated for local mod files. This needs now investigation but not worth at this moment.

Reply:
ok please create an issue so we don't forget